src/devrules/core/project_service.py

```
@@ -627,3 +652,57 @@ def _norm_status_key(value: Optional[str] = None) -> str:
+        return True if result.returncode == 0 else False
+    except Exception as e:
+        typer.secho(f"Error opening issue on web: {e}", fg=typer.colors.RED)
+        return False
+
+
+def get_issue_evidence(issue: str) -> list[str]:
+    """Get issue list of assets added as evidence"""
+    evidence: list[str] = []
+
+    try:
+        result = subprocess.run(
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)